lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +141,14 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($value instanceof OCI8Descriptor) {
```

Comment:
@soyuka yeah but your gist is not reflecting what Doctrine will be doing in the end because you are binding a descriptor to the `CLOB` column whereas Doctrine will not be using a descriptor.